src/app/dashboard/reports/page.tsx

```
@@ -50,123 +50,116 @@ export default function ReportsDashboard() {
             <LineChart className="text-indigo-500 h-5 w-5" />
           </div>
-          <p className="text-gray-600 mb-4">
+          <p className="text-gray-600 text-sm mb-4">
             Generate detailed sales reports by date range, product, or region.
           </p>
           <button
-            onClick={() => showModal('salesReportModal')} // <--- FIXED
-            className="bg-indigo-600 text-white px-4 py-2 rounded-md hover:bg-indigo-700 transition"
+            onClick={() => showModal('salesReportModal')}
+            className="w-full bg-indigo-600 text-white py-2 rounded-md text-sm hover:bg-indigo-700 transition"
           >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)